app/lib/conversation_list_pane/conversation_list.dart

```diff
@@ -162,17 +158,20 @@ class _ConversationListState extends State<ConversationList> {
 
     final coreClient = context.coreClient;
 
+    // TODO: Something is wrong here
     if (lastMessage != null) {
-      lastMessage.message.when(
-          contentFlight: (c) {
-            final lastContentMessage = c.last;
-            if (lastContentMessage.sender == coreClient.username) {
-              sender = 'You: ';
-            }
-            displayedLastMessage = lastContentMessage.content.body;
-          },
-          display: (d) => '',
-          unsent: (u) => '⚠️ Unsent message: ${u.body}');
```

Comment:
@raphaelrobert: Looks like this value is unused. Same for the `display` case. What is the intended way for this to work?

Reply:
Good catch. The value should be assigned to `displayedLastMessage` obviously.